netbox/dcim/tables/template_code.py

```diff
@@ -56,9 +56,13 @@
 
 INTERFACE_TAGGED_VLANS = """
 {% if record.mode == 'tagged' %}
+  {% if value.count > 3 %}
+    <a href="{{ record.get_absolute_url }}#vlans">{{ value.count }}</a>
```

Comment:
Instead of the interface view, let's link to the VLAN list with a filter for the current interface. This will be consistent with the way we currently handle assigned IP addresses (see `INTERFACE_IPADDRESSES`).

Reply:
That was also my initial plan, but VLANs doesn't seem to have interface filter like IP-address do.
So ipam/vlans/?(vm)interface_id=XXX needs to be implemented first for this to work.
That's why I implemented it with "vlan" anchor on interface page instead.

Reply:
Ah right. I've just opened FR #17669 to add filters for this. I'll see if I can knock that out real quick today.

Reply:
I've updated the pull request to support #17669